README.md

```diff
@@ -1 +1,41 @@
-# Hello-World
\ No newline at end of file
+Hello Microverse
+
+This is my first project.
+
```

Comment:
- [ ] [REQUIRED] I see that you're typing your README file by hand. This is why it has a lot of errors. Instead of doing this. Please copy the raw content of the microverse readme file as shown in the video below, then edit it as you like:

https://user-images.githubusercontent.com/52934128/164166841-54011b16-9c21-49df-837e-bc41aa47d34a.mp4